lib/ansible/modules/network/junos/junos_facts.py

```diff
@@ -76,12 +76,16 @@
   returned: always
   type: dict
 """
+
+import json
+
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils.junos import junos_argument_spec, check_args, get_param
 from ansible.module_utils.junos import get_configuration
 from ansible.module_utils.pycompat24 import get_exception
 from ansible.module_utils.netconf import send_request
 from ansible.module_utils.six import iteritems
+from ansible.module_utils._text import to_text
```

Comment:
Is `to_text` used anywhere in this module?

Reply:
Updated